alerts/cassandra/metadata.yaml

```diff
@@ -0,0 +1,33 @@
+alert_policy_templates:
+-
+  id: high-read-latency
+  display_name: Cassandra - High Read Latency
+  description: "Read latency is generally considered the slower operation in Cassandra\
```

Comment:
Also, consider using YAML multiline syntax:

    description: >-
      Read latency ...
      due to ...
      of how ...